libafl/src/executors/sand.rs

```diff
@@ -137,13 +149,19 @@ where
         let ot = self.executor.observers();
         let ob = ot.get(&self.ob_ref).unwrap().as_ref();
         let initial = ob.initial();
-        let covs = match self.pattern {
-            SANDExecutionPattern::SimplifiedTrace => ob
-                .as_iter()
-                .map(|x| if *x == initial { 0x1 } else { 0x80 })
-                .collect::<Vec<_>>(),
-            SANDExecutionPattern::UniqueTrace => ob.to_vec(),
-        };
+        let mut covs = ob.to_vec();
+        match self.pattern {
+            SANDExecutionPattern::SimplifiedTrace => {
+                for it in &mut covs {
+                    *it = if *it == initial { 0x1 } else { 0x80 };
```

Comment:
SIMD instructions can be used here. 
if you want to add you can, else can you add a `TODO: use SIMD instruction here`

Reply:
👍